frontend/app/mocks/db.ts

```diff
@@ -170,30 +170,30 @@ db.subscription.create({
 
 db.subscriptionConfirmationCode.create({
   id: '0000001',
-  email: 'unverified@example.com',
-  confirmationCode: '0001',
+  userId: '76c48130-e1d4-4c2f-8dd0-1c17f9bbb4f6',
+  code: '0001',
   createdDate: new Date(new Date().getTime() - 12 * 24 * 60 * 60 * 1000), // current date  date - 12 days
   expiryDate: new Date(new Date().getTime() - 10 * 24 * 60 * 60 * 1000), // current date  date - 10 days
 });
 db.subscriptionConfirmationCode.create({
   id: '0000002',
-  email: 'unverified@example.com',
-  confirmationCode: '1234',
+  userId: '76c48130-e1d4-4c2f-8dd0-1c17f9bbb4f6',
+  code: '1234',
   createdDate: new Date(new Date().getTime() - 4 * 24 * 60 * 60 * 1000),
-  expiryDate: new Date(new Date().getTime() - 2 * 24 * 60 * 60 * 1000), // current date date + 2 days
+  expiryDate: new Date(new Date().getTime() + 2 * 24 * 60 * 60 * 1000), // current date date + 2 days
 });
 db.subscriptionConfirmationCode.create({
   id: '0000003',
-  email: 'unverified@example.com',
-  confirmationCode: '1001',
+  userId: '76c48130-e1d4-4c2f-8dd0-1c17f9bbb4f6',
+  code: '1001',
   createdDate: new Date(new Date().getTime() - 8 * 24 * 60 * 60 * 1000), // current date date - 8 days
   expiryDate: new Date(new Date().getTime() - 6 * 24 * 60 * 60 * 1000), // current date date - 6 days
 });
 
 db.subscriptionConfirmationCode.create({
   id: '0000007',
-  email: 'tester@example.com',
-  confirmationCode: '2002',
+  userId: '8769695a4bcb78e132e06f96c246ea8e',
```

Comment:
Shouldn't this `userId` be an ID corresponding to an existing user (in UUID format)?

Reply:
fixed